zipline/assets/futures.py

```diff
@@ -126,15 +126,13 @@ def _maybe_update_current_chain(self):
             list
                 The up-to-date current chain, a list of Future objects.
         """
-        dt = self._get_datetime()
-
-        if (self._last_updated is None) or (self._last_updated != dt):
+        if (self._last_updated is None)\
+                or (self._last_updated != self.as_of_date):
```

Comment:
Yeah it will, unless `as_of_date` is specified, it will track the date of the simulation.
https://github.com/quantopian/zipline/blob/future-chain-start-date/zipline/assets/futures.py#L105